CHANGELOG.md

```diff
@@ -86,6 +86,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * [#13236](https://github.com/cosmos/cosmos-sdk/pull/13236) Integrate Filter Logging
 * [#13528](https://github.com/cosmos/cosmos-sdk/pull/13528) Update `ValidateMemoDecorator` to only check memo against `MaxMemoCharacters` param when a memo is present.
 * [#13651](https://github.com/cosmos/cosmos-sdk/pull/13651) Update `server/config/config.GetConfig` function.
+* [#13780](https://github.com/cosmos/cosmos-sdk/pull/13780) `id` (type of int64) in `AccountAddressByID` grpc query is now deprecated, update to use account-id(type of uint64) to request `AccountAddressByID`.
```

Comment:
This should go in  the state-machine breaking section

Reply:
didn't understand, why this should go under state-machine breaking? we are updating only a query request field.

Reply:
It's state-machine breaking since we added the deterministic annotation!